static/app/components/profiling/flamegraphSearch.tsx

```diff
@@ -321,13 +327,45 @@ function FlamegraphSearch({
     <StyledSearchBar
       size="xs"
       placeholder={t('Find Frames')}
-      query={search.query}
+      defaultQuery={search.query}
       onChange={handleChange}
       onKeyDown={handleKeyDown}
+      trailing={
+        search.query && (
+          <Fragment>
+            <StyledTrailingText>
+              {`${
+                search.index !== null && search.results.size > 0 ? search.index + 1 : '-'
+              }/${search.results.size}`}
+            </StyledTrailingText>
+            <SearchBarTrailingButton
+              type="button"
+              size="zero"
+              borderless
+              icon={<IconChevron size="xs" />}
+              aria-label={t('Next')}
+              onClick={onNextSearchClick}
+            />
+            <SearchBarTrailingButton
+              type="button"
+              size="zero"
+              borderless
+              icon={<IconChevron size="xs" direction="down" />}
+              aria-label={t('Previous')}
+              onClick={onPreviousSearchClick}
+            />
+          </Fragment>
+        )
+      }
```

Comment:
@getsentry/core-ui  not sure if you would see this becoming a permanent option in the `SearchBar` 
![image](https://user-images.githubusercontent.com/7349258/202803790-1664e0de-90ed-4133-8856-cfc47a8e66aa.png)


for now I've just allowed us to add it via a `trailing` prop